components/serverless/internal/testenv/testenv.go

```diff
@@ -0,0 +1,54 @@
+package testenv_test
```

Comment:
I wonder if the `testenv_test` keeps redundant info. The `_test` suffix is used to define the desire of the package, right? I would suggest renaming it into `env_test`. 

To be honest, I think the `testenv` keeps all important information about the content under it, so maybe we even should remove this _test suffix.

But if we don't want to allow importing public methods from this pkg in a non-test file we should add the `_test` suffix to the file name.

Reply:
It's not possible to use test file package in other tests I renamed the file to just `envtest`. 
I removed the additional `test` from package as `Start` and `Stop` accept `testing.T` which indicate that those methods should be used in test only,